src/reference/crash-reporting.md

```diff
@@ -25,19 +25,29 @@ over time. A crash report may contain the following information:
 Google handles all data reported by this tool in accordance with the 
 [Google Privacy Policy][].
 
+Additionally, you may review the recently reported data in the 
+`.dart-tool/dart-flutter-telemetry.log` file under the home directory
+`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory. 
+
 ## Disabling analytics reporting
 
 You can opt out of anonymous crash reporting and feature
 usage statistics from Flutter by running the following command:
 
 ```terminal
-$ flutter config --no-analytics
+$ flutter --disable-telemetry
 ```
 
 If you opt out of analytics, an opt-out event will be sent,
-and then no further information will be sent by that
+and then no further information will be stored or sent by that
 installation of Flutter. 
```

Comment:
**Issue:** Split sentences to ease readability.
```suggestion
If you opt out of analytics, Flutter sends an opt-out event.
This Flutter installation neither sends nor stores any further information. 
```

Reply:
Committed the change!